client/src/main/scala/io/delta/sharing/client/DeltaSharingFileSystem.scala

```diff
@@ -69,7 +70,31 @@ private[sharing] class DeltaSharingFileSystem extends FileSystem with Logging {
       val proxy = new HttpHost(proxyConfig.host, proxyConfig.port)
       clientBuilder.setProxy(proxy)
 
```

Comment:
below is a `HttpRequestExecutor` that is conditionally (spark config) set to the client that effectively downgrades https requests to http by returning a  `wrappedRequest` with a modified uri. 
- new code path is only activated by spark config
- any failures during downgrade are swallowed and a log is emitted